openqaoa/devices.py

```diff
@@ -256,6 +266,112 @@ def check_connection(self) -> bool:
         """
 
         return True
+    
+    
+class DeviceAWS(DeviceBase):
+    
+    """
+    Contains the required information and methods needed to access QPUs hosted
+    on AWS Braket.
+    
+    Attributes:
+	available_qpus: `list`
+		When connection to AWS is established, this attribute contains a list
+		of device names which can be used to access the selected device by
+		reinitialising the Access Object with the name of the available device
+		as input to the device_name parameter.
+    """
+    
+    def __init__(self, device_name: str, aws_access_key_id: str, 
+                 aws_secret_access_key: str, aws_region: str, 
+                 s3_bucket_name: str, folder_name: str = 'oq_runs'):
```

Comment:
Let's call it `openqaoa` if we set a standard. Matching `oq` to `openqaoa` is both trivial and impossible at the same time!